tests/store/tracking/test_file_store.py

```diff
@@ -49,6 +50,8 @@ def create_test_run(self):
     def create_experiments(self, experiment_names):
         ids = []
         for name in experiment_names:
+            # ensure that the field `last_update_time` is distinct for search ordering
+            time.sleep(0.05)
```

Comment:
```suggestion
            time.sleep(0.001)
```

Can we shorten the sleep length?

Reply:
yep!